docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -49,8 +49,9 @@ export const HeroSection = () => {
           alignItems="center"
         >
           <Heading level={1} marginBlockEnd="0">
-            Supercharge your app &mdash; <br />
-            <strong>Connect your UI to the cloud.</strong>
+            Themeable, accessible components
```

Comment:
Thinking that we did not write this but should there be a period or a dash at the end of this line to indicate the end of this line since we capitalize `Themeable` on the next? Also, since we are currently using `themable` in the rest of the docs ([example](https://ui.docs.amplify.aws/react/getting-started/accessibility#themable-design-tokens)), should we either align this usage with the current spelling, or update the existing usage to  `themeable`?

Reply:
Or `theme-able` which is what is most used on the site currently when I use the site search (I think I introduced `themable` as an outlier, oops 😉)